src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  const declarations = sourceString.split(';');
+
+  declarations.forEach((declaration) => {
+    const trimmedDeclaration = declaration.trim();
+
+    if (!trimmedDeclaration) {
+      return;
+    }
+
+    const [property, value] = trimmedDeclaration
+      .split(':')
+      .map((part) => part.trim());
+
+    stylesObject[property] = value;
```

Comment:
[CODE PRACTICE] - Instead of using a `forEach` loop to iterate over the declarations, consider using `reduce` to build the styles object. This approach aligns with the task's requirement to avoid loops and use iteration methods more effectively.